factory/consensus/consensusComponents.go

```diff
@@ -667,6 +672,19 @@ func (ccf *consensusComponentsFactory) createPeerBlacklistHandler() (consensus.P
 	return blacklist.NewPeerBlacklist(blacklistArgs)
 }
 
+func (ccf *consensusComponentsFactory) createP2pSigningHandler() (consensus.P2PSigningHandler, error) {
+	p2pSignerArgs := &p2pFactory.ArgsMessageVerifier{
```

Comment:
You can remove & from here and * from L680

Reply:
done